File: capybara_rspec/remote_webdriver/spec/test_scenario/example_test.rb
require_relative '../spec_helper'

feature 'TrueAutomation.IO Capybara example' do
  scenario 'Test example' do
    visit 'https://app.trueautomation.io/'

    find(:xpath, ta('homePage:loginBtn', "//a[.//span[text()='Login']]")).click
    find(:css, ta('signinPage:signupBtn', 'div.sign-up-container > a')).click
    fill_in ta('signupPage:emailField', 'email'), with: 'your@mail.com'
  end
end

File: javascript_wdio/waits/test/specs/test_scenario/exampleTest.js
const { ta } = require('trueautomation-helper');

describe('TrueAutomation.IO WebdirverIO example', () => {
  beforeEach(function() {
  });

  it('Test example (not run)', () => {
    browser.url('https://trueautomation.io');
    $(ta('homePage:loginBtn', "//a[.//span[text()='Login']]")).click();
    // $(ta('signinPage:signupBtn', 'div.sign-up-container > a')).click();
    // $(ta('signupPage:emailField', "[name='email']")).setValue('your@gmail.com');
  });

  it('Test example', () => {
    browser.url('https://trueautomation.io');
    $(ta('homePage:loginBtn', "//a[.//span[text()='Login']]")).click();
    // $(ta('signinPage:signupBtn', 'div.sign-up-container > a')).click();
    // $(ta('signupPage:emailField', "[name='email']")).setValue('your@gmail.com');
  });
});
